fpd/fpd_bmc_bios.cc

```diff
@@ -206,7 +240,14 @@ Fpd_bmc_bios::running_version(void) const
 
     helper.push_back(helper_script);
     fpd_t::invoke(helper, arg_list);
-    return get_bios_version_from_spiflash();
+
+    select_mux();
```

Comment:
this code does not seem to work correctly if multiple invocations are trying to obtain the version, or one is trying to get a version and one is upgrading, or if multiple invocations are trying to upgrade.

Reply:
Same as earlier, have created an internal JIRA to track this:
MIGSOFTWAR-4736

Reply:
Jira opened for this issue.